### PR TITLE
Rename risk metadata vpnServiceName to hostingName

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Summary
Rename risk metadata vpnServiceName to hostingName because `isHosting` related to that field

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Review by nanobash — DISMISSED** [review]

**Review by lexerom — DISMISSED** [review]

**Review by noon-dawg — DISMISSED** [review]

**Review by removed-account-123 — DISMISSED** [review]

File: openapi/components/schemas/RiskMetadata.yaml [hunk 1]
```diff
@@ -114,8 +114,8 @@ properties:
     description: Specifies if the customer's IP address is related to hosting.
     type: boolean
     readOnly: true
-  vpnServiceName:
-    description: VPN service name, if available.
+  dch:
```
Comment (ashkarpetin) [inline openapi/components/schemas/RiskMetadata.yaml:117]:
```suggestion
  hostingName:
```
isHosting is related to this field, so I wonder if we should have consistent naming?

File: openapi/components/schemas/RiskMetadata.yaml [hunk 2]
```diff
@@ -114,8 +114,8 @@ properties:
     description: Specifies if the customer's IP address is related to hosting.
     type: boolean
     readOnly: true
-  vpnServiceName:
-    description: VPN service name, if available.
+  hostingName:
+    description: Name of the data center or hosting provider, if available.
```
Comment (ashkarpetin) [inline openapi/components/schemas/RiskMetadata.yaml:118]: Changed name from `dch` to the `hostingName` because `isHosting` related to that field

**Comment by ashkarpetin** [comment]
php please re-review

**Comment by ashkarpetin** [comment]
js please re-review

**Review by colinbird — APPROVED** [review]

**Review by dara123 — APPROVED** [review]

**Review by wmdmgv — APPROVED** [review]

**Pull request merged**